contributing/code_of_conduct/index.rst

```diff
@@ -0,0 +1,91 @@
+Code of Conduct
+===============
+
+Our Pledge
+----------
+
+In the interest of fostering an open and welcoming environment, we as
+contributors and maintainers pledge to making participation in our project and
+our community a harassment-free experience for everyone, regardless of age, body
+size, disability, ethnicity, gender identity and expression, level of experience,
```

Comment:
After the last change, `ethnicity` and `ethnic origin` are used in the same paragraph. We should remove `ethnicity` and keep just `ethnic origin`.